content/overview/security/fedramp-tracker.md

```diff
@@ -12,30 +12,24 @@ aliases:
 ---
 
 
-cloud.gov has a [Provisional Authority to Operate (P-ATO) at the Moderate impact level from the FedRAMP Joint Authorization Board (JAB)](https://marketplace.fedramp.gov/#/product/18f-cloudgov).
+cloud.gov has a [Provisional Authority to Operate (P-ATO) at the Moderate impact level from the FedRAMP Joint Authorization Board (JAB)](https://marketplace.fedramp.gov/#/product/18f-cloudgov). This means cloud.gov has undergone a significant, thorough security and compliance review so that your agency doesn't have to.
 
-## How this P-ATO helps your team
+## What is a P-ATO?
 
-This P-ATO represents a comprehensive security and compliance assessment that enables federal agencies to start using cloud.gov with significantly reduced effort.
+The **[Federal Risk and Authorization Management Program (FedRAMP)](https://www.fedramp.gov/)** evaluates cloud services and issues a **Provisional Authority to Operate (P-ATO)** to those that pass review. Those come in two flavors: Agency and JAB. Both authorizations [look at a standardized set of requirements in accordance with FISMA](https://www.fedramp.gov/jab-or-agency-how-do-i-get-a-fedramp-ato/) but JAB P-ATOs approve cloud services for governmentwide operation. The **Joint Authorization Board (JAB)** is the panel of the CIOs of the General Services Administration, Department of Defense, and Department of Homeland Security that issues [these kinds of authorizations](https://marketplace.fedramp.gov/#/products?status=Compliant&sort=productName&authorizationType=JAB). **Authorization at the moderate level** means cloud.gov is a vetted and trustable service for any government system that contains data where [the impact of loss is limited or serious — but not catastrophic](http://csrc.nist.gov/publications/fips/fips199/FIPS-PUB-199-final.pdf#page=6).
```

Comment:
Agency P-ATOs also enable use of cloud services in other agencies, not just JAB P-ATOs. We should be precise with how we describe FedRAMP so that we're in line with their messaging. A helpful thing about the previous language is that FedRAMP already checked it as part of the post at https://18f.gsa.gov/2017/02/02/cloud-gov-is-now-fedramp-authorized/ - we can ask FedRAMP to take a look again though.

Reply:
I was trying to adapt the language on [FedRAMP's page](https://www.fedramp.gov/jab-or-agency-how-do-i-get-a-fedramp-ato/) for this. Seems like the difference is that the JAB review is done with a governmentwide scope whereas the agency review is done only for that specific agency. My understanding is that an agency could accept another agency's Agency P-ATO but they'd have more confidence with a JAB P-ATO.